src/main/drivers/vtx_common.h

```diff
@@ -81,6 +83,9 @@ typedef enum {
 #define VTX_TRAMP_POWER_200       3
 #define VTX_TRAMP_POWER_400       4
 #define VTX_TRAMP_POWER_600       5
+#define VTX_TRAMP_POWER_MAX       5
+
+#define VTX_POWER_MAX             MAX(VTX_6705_POWER_MAX, VTX_SA_POWER_MAX, VTX_TRAMP_POWER_MAX)
```

Comment:
Not sure what this is intended for - it does not seem to be used, and `VTX_TABLE_MAX_POWER_LEVELS` should be used anyway.

Reply:
The intent was to minimise the size of structures, but thinking about it, I should be setting `VTX_SA_POWER_MAX` to  `VTX_TABLE_MAX_POWER_LEVELS` to allow support for more power levels should a VTX manufacturer provide them.